.envrc

```diff
@@ -16,4 +16,6 @@ HOOK_TYPES=(pre-commit pre-merge-commit pre-push prepare-commit-msg commit-msg p
 for HOOK_TYPE in ${HOOK_TYPES[@]}; do pre-commit uninstall -t "$HOOK_TYPE" > /dev/null; done
 
 # install pre-commit hook (opt-out by setting `DADE_NO_PRE_COMMIT`, set the hook type with `DADE_PRE_COMMIT_HOOK_TYPE` -- defaults to 'pre-push')
-test "x$DADE_NO_PRE_COMMIT" = x && pre-commit install -t "$DADE_PRE_COMMIT_HOOK_TYPE" > /dev/null
+if [[ -z "${DADE_NO_PRE_COMMIT:-}" ]]; then
```

Comment:
I fixed this en-passant. Locally, `direnv` (2.32.2), fails on this line because it cannot resolve `DADE_NO_PRE_COMMIT`. Since the point of this is to test its existence, I gave it a default empty value.

An `if` seemed easier to read for a wider audience than using `test` with a prefix.

Reply:
Still _really_ not fond of that being opt-out.